zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -138,6 +138,44 @@ public void testCreateNewButton() throws InterruptedException {
   }
 
   @Test
+  public void testWidth() throws InterruptedException {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+
+      collector.checkThat("Default Width is 12 ",
+              driver.findElement(By.xpath("//div[contains(@class,'col-md-12')]")).isDisplayed(),
+              CoreMatchers.equalTo(true));
+      for (Integer newWidth = 1; newWidth <= 11; newWidth++) {
+        driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+        String visibleText = newWidth.toString();
+        new Select(driver.findElement(By.xpath(getParagraphXPath(1)
+                + "//ul/li/a/form/select[(@ng-change='changeColWidth()')]"))).selectByVisibleText(visibleText);
+        collector.checkThat("New Width is : " + newWidth,
+                driver.findElement(By.xpath("//div[contains(@class,'col-md-" + newWidth + "')]")).isDisplayed(),
+                CoreMatchers.equalTo(true));
+      }
+
+
+      ZeppelinITUtils.sleep(1000, false);
+
+      deleteTestNotebook(driver);
+
+    } catch (Exception e) {
+      LOG.error("Exception in ParagraphActionsIT while testWidth ", e);
+      File scrFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+      throw e;
```

Comment:
throw e (Exception);  but signature throws InterruptedException, can you change that ?